core/server/api/shared/frame.js

```diff
@@ -37,6 +37,10 @@ class Frame {
             }
         }
 
+        if (apiConfig.rawInclude) {
```

Comment:
https://github.com/TryGhost/Ghost/pull/10051#discussion_r227787385 still applies to this part of code